modules/core/client/config/core.client.routes.js

```diff
@@ -15,28 +19,52 @@ angular.module('core').config(['$stateProvider', '$urlRouterProvider',
     $stateProvider
     .state('home', {
       url: '/',
-      templateUrl: 'modules/core/client/views/home.client.view.html'
+      views: {
+        'hero': {
+          templateUrl: 'modules/core/client/views/hero.client.view.html'
+        },
+        'main': {
+          controller: 'HomeController',
```

Comment:
totally want to, but didn't want to introduce it in this PR...  If I go that route I would need to do it for all the routes which means I have to rewrite almost the entire user client side I think.  Maybe we should introduce that in another PR to get it up to date with our standards?